src/main/java/practice/StreamPractice.java

```
@@ -23,7 +40,14 @@
      * But before that subtract 1 from each element on an odd position (having the odd index).
      */
     public Double getOddNumsAverage(List<Integer> numbers) {
-        return 0D;
+        if (numbers == null || numbers.isEmpty()) {
+            return 0D;
```

Consider throwing a `NoSuchElementException` instead of returning 0D when the list is null or empty, to maintain consistency with other methods that throw exceptions when no valid data is found.